blocks/columns/columns.js

```diff
@@ -1,33 +1,46 @@
 import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
-import { div } from '../../scripts/scripts.js';
+import { div, span } from '../../scripts/scripts.js';
+
+/* ------------------------------ Global Variables ----------------------------------- */
+// media query match that indicates mobile/tablet width
+const isMobile = window.matchMedia('(max-width: 768px)');
```

Comment:
When I start on a big window, and then resize it to make it small, the button does not work.

Reply:
Yeah I setup the click handlers to only be registered if the isMobile MediaQuery matches (so less than 768px) but that really doesn't make sense given the exact use case you've described.  I'll remove that limitation so that the click handlers will be there if the buttons are displayed due to a resize of the screen. 

Thanks!

Reply:
Donezo, have another look and let me know what you think  :D